network/book_test.go

```diff
@@ -30,19 +31,15 @@ func TestAddSavesPeer(t *testing.T) {
 
     entry, _ := book.Get()
 
-    if entry != addr {
-        t.Fatalf("Entry %s was not found in book", addr)
-    }
+    assert.Equal(t, addr, entry, "Entry %s was not found in book", addr)
 }
 
 func TestGetReturnsErrBookEmpty(t *testing.T) {
     book := NewSimpleBook()
 
     _, err := book.Get()
 
-    if err != errBookEmpty {
-        t.Fatalf("Get should return error in case book is empty")
-    }
+    assert.Equal(t, errBookEmpty, err, "Get should return error in case book is empty")
```

Comment:
One thing I missed last time: different error types should not be used programmatically unless they are exported (in which case they are called sentinel errors). You should also never use the error message to check for a specific error. So in this case, I would simply put `err == nil`.